components/globals/Alert/Alert.tsx

```diff
@@ -195,7 +195,7 @@ const AlertContainer = ({
     <div
       ref={refFocus}
       {...(focussable && { tabIndex: -1 })}
-      className={cn("relative flex rounded-lg p-4", className)}
+      className={cn("relative flex laptop:rounded-md rounded-sm p-4", className)}
```

Comment:
I think you want to flip these- ie, default size for small screen sizes comes first, then whatever breakpoint you want to change it at for that size and larger. so it should be `rounded-sm laptop:rounded-md`